third_party/nxp/k32w0_sdk/sdk_fixes/patch_k32w_sdk.sh

```diff
@@ -8,19 +8,23 @@ fi
 SOURCE=${BASH_SOURCE[0]}
 SOURCE_DIR=$(cd "$(dirname "$SOURCE")" >/dev/null 2>&1 && pwd)
 
-cp ./third_party/nxp/k32w0_sdk/sdk_fixes/gpio_pins.h "$NXP_K32W061_SDK_ROOT"/boards/k32w061dk6/wireless_examples/openthread/reed/bm/
-cp ./third_party/nxp/k32w0_sdk/sdk_fixes/pin_mux.c "$NXP_K32W061_SDK_ROOT"/boards/k32w061dk6/wireless_examples/openthread/enablement/
+dos2unix "$NXP_K32W061_SDK_ROOT"/boards/k32w061dk6/wireless_examples/openthread/reed/bm/gpio_pins.h
```

Comment:
how about checking in the files as unix  to start with (run dos2unix on the windows machine if you are  using that or in docker)?

This way you do not need to change the ble_utils_h.patch.

Reply:
Good idea, let me check this.